packages/logs/src/boot/startLogs.ts

```diff
@@ -87,6 +90,11 @@ export function doStartLogs(
     )
   }
   errorObservable.subscribe(reportError)
+  if (isExperimentalFeatureEnabled('forward-logs')) {
+    trackConsoleLogs(['log', 'info', 'debug', 'warn'], (log: ConsoleLog) => {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** type can be inferred‏
```suggestion
    trackConsoleLogs(['log', 'info', 'debug', 'warn'], (log) => {
```